README.md

```diff
@@ -1,104 +1,113 @@
+
+- For the list of operator dependencies, see the IBM Knowledge Center [Common Services dependencies documentation](http://ibm.biz/cpcs_opdependencies).
+- For the list of prerequisites for installing the operator, see the IBM Knowledge Center [Preparing to install services documentation](http://ibm.biz/cpcs_opinstprereq).
+
+## Documentation
+
+To install the operator with the IBM Common Services Operator follow the the installation and configuration instructions within the IBM Knowledge Center.
+
+- If you are using the operator as part of an IBM Cloud Pak, see the documentation for that IBM Cloud Pak. For a list of IBM Cloud Paks, see [IBM Cloud Paks that use Common Services](http://ibm.biz/cpcs_cloudpaks).
+- If you are using the operator with an IBM Containerized Software, see the IBM Cloud Platform Common Services Knowledge Center [Installer documentation](http://ibm.biz/cpcs_opinstall).
+
+## SecurityContextConstraints Requirements
+
+The IBM System  Management ingress service now don't support running with the OpenShift Container Platform 4.3 default restricted Security Context Constraints (SCCs).
+
+For more information about the OpenShift Container Platform Security Context Constraints, see [Managing Security Context Constraints](https://docs.openshift.com/container-platform/4.3/authentication/managing-security-context-constraints.html).
 
-### PodSecurityPolicy Requirements
+## Developer guide
 
-### SecurityContextConstraints Requirements
+If, as a developer, you are looking to build and test this operator to try out and learn more about the operator and its capabilities, you can use the following developer guide. This guide provides commands for a quick install and initial validation for running the operator.
 
-The management service supports running under the OpenShift Container Platform default restricted security context constraints.
+> **Important:** The following developer guide is provided as-is and only for trial and education purposes. IBM and IBM Support does not provide any support for the usage of the operator with this developer guide. For the official supported install and usage guide for the operator, see the the IBM Knowledge Center documentation for your IBM Cloud Pak or for IBM Cloud Platform Common Services.
 
-## Getting Started
+### Quick start guide
 
-### Cloning the repository
+Use the following quick start commands for building and testing the operator:
 
-Checkout this Management Ingress Operator repository
+#### Cloning the repository
+
+Check out the ibm-management-ingress-operator repository.
 
 ```bash
 # git clone https://github.com/IBM/ibm-management-ingress-operator.git
 # cd ibm-management-ingress-operator
 ```
 
-### Building the operator
+#### Building the operator
 
-Build the meta operator image and push it to a public registry, such as quay.io:
+Build the ibm-management-ingress-operator image and push it to a public registry, such as quay.io.
```

Comment:
Minor edit (capitalize Quay)

Build the ibm-management-ingress-operator image and push it to a public registry, such as Quay.io.

Reply:
Updated.